os/adding-swap.md

```diff
@@ -0,0 +1,95 @@
+# Adding swap in CoreOS
+
+Swap is the process of moving pages of memory to a designated part of the hard disk, freeing up space when needed. Swap can be used to alleviate problems with low-memory environments.
+
+By default CoreOS does not include a partition for swap, however one can configure their system to have swap.
+
+## Managing swap with systemd
+
+Systemd can be used to create, begin, and stop a swapfile on your system. Write the following systemd unit file and enable + start the service to add swap to your instance of CoreOS.
+
+### Creating the systemd unit file
+
+The following systemd unit file creates, starts, and stops a 1GiB swapfile. It should be written to `/etc/systemd/system/swap.service`.
+
+```
+[Unit]
+Description=Turn on swap
+
+[Service]
+Type=oneshot
+Environment="SWAP_PATH=/var/vm" "SWAP_FILE=1GiB.swap"
```

Comment:
change: SWAP_FILE=swapfile1 so that it matches the ignition version of the same thing, below